tests/base.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+import random
+import string
+
+from os.path import expanduser
+
+
+FILE = 'token.json'
+AUTH_TOKEN_PATH = "{}/.evalai/".format(expanduser('~'))
+
+
+class BaseTest:
+
+    @classmethod
+    def setup_class(cls):
+        if not os.path.exists(AUTH_TOKEN_PATH):
+            os.makedirs(AUTH_TOKEN_PATH)
+            with open(AUTH_TOKEN_PATH + FILE, 'w') as fw:
+                token = ''.join(random.choice(string.ascii_lowercase) for _ in range(40))
```

Comment:
Use `random.choices(string.ascii_lowercase, k=40)` to make this line more terse and easier to read.
Also, you may want to set a seed for the PRNG in order to make this reproducible.

Reply:
Only thing is, `random.choices` is only for Python 3.6 and it's not backward compatible. So I should stick with this, yeah?

Reply:
Ah I see. Good catch.